### DISCUSSION TITLE
Python issue

### DISCUSSION THREAD

**Discussion by DrSolver98** [doocs/leetcode#3279] [General] [answered]
Hi there,
I'm new to python and have a problem with module installation via pip.
Python version: 2.6, OS: Mac OS Sonoma
Error: "bash: pip: command not found"
>pip: command not found

**Comment by VyvaHart** [comment, marked as answer]
Hi @DrSolver98 

Try: `apt-get install python3-pip` to install pip

Good Luck!

**Comment by Pushkar-Saini** [comment]
Try: apt-get install python3-pip to install pip
